### ISSUE TITLE
Appears that `nginx` might not be available on the `$PATH`

### ISSUE THREAD

**Issue by ryanmoran** [paketo-buildpacks/nginx#203]
## What happened?

* What were you attempting to do?
Writing a wrapper script around `nginx` to do something before running the server:

  ```shell
  # set PATH
  export PATH=$PATH:/layers/paketo-buildpacks_nginx/nginx/sbin

  cat /workspace/nginx.conf

  nginx -p /workspace -c /workspace/nginx.conf
  ```

* What did you expect to happen?
I expected not to need to set the `$PATH`.

* What was the actual behavior? Please provide log output, if possible.
I needed to set the `$PATH` so that `nginx` could be found in my script.

Details come from this [blog post](https://griggheo.medium.com/using-the-paketo-nginx-buildpack-to-build-an-nginx-docker-image-42f4dc401df8).

## Checklist

* [x] I have included steps for reproduction.

**Comment by arjun024** [comment]
Couldn't reproduced. Feel free to reopen if issue persists.

**Comment by aliakbar-hemmati** [comment]
I have the same issue, too. It seems the directory that contains nginx binary is not in the PATH.

**Comment by arjun024** [comment]
@aliakbar-hemmati
The nginx executable is located at `/<layers-directory>/paketo-buildpacks_nginx/nginx/sbin` which is added to
$PATH [here](https://github.com/paketo-buildpacks/nginx/blob/v0.3.1/build.go#L137). 

Could you please provide more details? What's the behavior you are seeing.?

**Comment by aliakbar-hemmati** [comment]
@arjun024 
Yes, I have seen the message of it in build logs. But I built the image using paketo static buildpack and when I run the container, it exits with code 0. When I provide this command `nginx -p /workspace -c /workspace/nginx.con` for it to start with it says it cannot find the file. This is the command:

```
/ # docker run -p80:80 my_image_name nginx -p $PWD -c "/workspace/nginx.conf"
```

and output:

```
nginx -p $PWD -c "/workspace/nginx.conf": nginx -p /workspace -c /workspace/nginx.conf: No such file or directory
```

To investigate more I started it with `sleep` and got a shell into it. When I run `echo $PATH` the output is like this:  
```
/cnb/process:/cnb/lifecycle:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```
It seems that nginx binary is not in PATH.

**Comment by arjun024** [comment]
If you are building using the nginx buildpack combined with the [staticfile community buildpack](https://github.com/paketo-community/staticfile) - then you don't have to provide a start command. It runs nginx as a start-command.

E.g. (this [app](https://github.com/paketo-community/staticfile/tree/v0.1.0/integration/testdata/nginx_helloworld))
```sh
pack build yourimage -b gcr.io/paketo-buildpacks/nginx -b gcr.io/paketo-community/staticfile
docker run --env PORT=8080 -p8080:8080 yourimage
```
(For documentation on `$PORT` see [docs](https://paketo.io/docs/howto/web-servers/#port))

To inspect if nginx is on PATH, you can do:
```sh
docker run --init -it --entrypoint=launcher yourimage 'command -v nginx'
/layers/paketo-buildpacks_nginx/nginx/sbin/nginx
```

You should always run your CNB produced image via the `launcher` entrypoint. See spec https://github.com/buildpacks/spec/blob/main/platform.md#launch.

Does that help?

**Comment by aliakbar-hemmati** [comment]
Thanks a lot. I will check it.

**Comment by aliakbar-hemmati** [comment]
Now I know what happened. I used my own `nginx.conf` and it did not include `daemon off;`. Because of that the container exits with code 0. I added that line to config and now it is working. Thank you @arjun024

**Comment by aliakbar-hemmati** [comment]
@ryanmoran 
I have not access to your blog post. Please check if adding `daemon off` works for you.

**Comment by ryanmoran** [comment]
@aliakbar-hemmati the original report came from a blog post that I did not write, but just found. In it, the writer was doing a bunch of extra work to get their app using `nginx` to work. Based on what they were doing, I reported that we might have a bug, but on further discussion with the writer, we discovered that they were not running the container in the expected way which resulted in some unexpected behavior. Here is the original thread on Slack: https://paketobuildpacks.slack.com/archives/CU8RVQZ1R/p1616084489004700.
